testkit/src/integration-testkit.ts

```diff
@@ -175,7 +175,7 @@ export class IntegrationTestkit {
         'VERSION_CHECK_ON_STARTUP',
         process.env.VERSION_CHECK_ON_STARTUP || 'true',
       )
-      .withWaitStrategy(Wait.forLogMessage('Starting Kalix '))
+      .withWaitStrategy(Wait.forLogMessage('Connecting to '))
```

Comment:
Seems the order of discovery and logs changed somewhat in the runtime, causing a deadlock because waiting for the `Starting Kalix ` log output, but that was only logged once discovery completes, and discovery would not complete until the test container proxy port is injected a few lines down from here.

Reply:
that's odd, the line is still there...

Reply:
I mean in the runtime code